### PR TITLE
SC data release notes 21

### PR DESCRIPTION
**Pull request by pmb59**
Starting release notes for September '24 SCXA release.

### CODE REVIEW

File: release-notes/sc/_posts/2024-09-14-21.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#### Data statistics
+
+- Ensembl **104** / Ensembl Genomes **51** / WormBase ParaSite **15** gene annotations.   
+- This release contains **381** single cell RNA-Seq studies, consisting of **24,323,556** cells, of which **13,519,191** passed our QC from **21** different species.
+
+#### New experiments
+- [Human Lung Cell Atlas: An integrated cell atlas of the human lung in health and disease](https://www.ebi.ac.uk/gxa/sc/experiments/E-ANND-1)
```
Comment (anilthanki) [inline release-notes/sc/_posts/2024-09-14-21.md:20]:
```suggestion
Externally analysed dataset
- [Human Lung Cell Atlas: An integrated cell atlas of the human lung in health and disease](https://www.ebi.ac.uk/gxa/sc/experiments/E-ANND-1)
```

File: release-notes/sc/_posts/2024-09-14-21.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+- [GTEx: snRNAseq atlas](https://www.ebi.ac.uk/gxa/sc/experiments/E-ANND-2)
+- [Tabula Sapiens - Smart-seq2 dataset](https://www.ebi.ac.uk/gxa/sc/experiments/E-ANND-3)
+- [Mapping the developing human immune system across organs](https://www.ebi.ac.uk/gxa/sc/experiments/E-ANND-5)
+- [Single cell RNA-sequencing of normal and fibrotic lungs in human](https://www.ebi.ac.uk/gxa/sc/experiments/E-CURD-126)
```
Comment (anilthanki) [inline release-notes/sc/_posts/2024-09-14-21.md:26]:
```suggestion

Dataset reanalysed with our standard pipelines
- [Single cell RNA-sequencing of normal and fibrotic lungs in human](https://www.ebi.ac.uk/gxa/sc/experiments/E-CURD-126)
```

**Comment by anilthanki** [comment]
Not sure if we should mention about an experiment we dropped `E-MTAB-6818`

File: release-notes/sc/_posts/2024-09-14-21.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+#### Data statistics
+
+- Ensembl **104** / Ensembl Genomes **51** / WormBase ParaSite **15** gene annotations.   
+- This release contains **381** single cell RNA-Seq studies, consisting of **24,323,556** cells, of which **13,519,191** passed our QC from **21** different species.
+
+#### Software features
+- **Gut anatomogram** - a new human anatomogram for the large and small intestines has been added in this release and accompanies eligible experiments e.g. [here](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-125970/results/anatomogram)
```
Comment (lingyun1010) [inline release-notes/sc/_posts/2024-09-14-21.md]: I think we could replace `here` with the experiment accession.

File: release-notes/sc/_posts/2024-09-14-21.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+- **Inferred cell type marker gene heatmap** - for experiments where inferred cell type data is available you can now view marker genes based on the inferred cell type clusters e.g. [here](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-227038/results/marker-genes)
+
+#### New experiments
+Highlights
+- **Externally analysed data** - for the first time in this release we have included a new class of data to Single Cell Expression Atlas - externally analysed data, marked with an 'E' icon. 
+  These are currently limited to a careful selection of eligible high-impact human datasets including the [GTEx: snRNAseq atlas](https://www.ebi.ac.uk/gxa/sc/experiments/E-ANND-2) As these    experiments are not re-analysed with our own analysis pipelines, not all the standard chart or data options may be available
```
Comment (lingyun1010) [inline release-notes/sc/_posts/2024-09-14-21.md]: Extra space between these and experiments

File: release-notes/sc/_posts/2024-09-14-21.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
+#### Data statistics
+
+- Ensembl **104** / Ensembl Genomes **51** / WormBase ParaSite **15** gene annotations.   
+- This release contains **381** single cell RNA-Seq studies, consisting of **24,323,556** cells, of which **13,519,191** passed our QC from **21** different species.
+
+#### Software features
+- **Gut anatomogram** - a new human anatomogram for the large and small intestines has been added in this release and accompanies eligible experiments e.g. [here](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-125970/results/anatomogram)
+- **Inferred cell type marker gene heatmap** - for experiments where inferred cell type data is available you can now view marker genes based on the inferred cell type clusters e.g. [here](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-227038/results/marker-genes)
```
Comment (lingyun1010) [inline release-notes/sc/_posts/2024-09-14-21.md]: Same comment for here

File: release-notes/sc/_posts/2024-09-14-21.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+- Ensembl **104** / Ensembl Genomes **51** / WormBase ParaSite **15** gene annotations.   
+- This release contains **381** single cell RNA-Seq studies, consisting of **24,323,556** cells, of which **13,519,191** passed our QC from **21** different species.
+
+#### Software features
+- **Gut anatomogram** - a new human anatomogram for the large and small intestines has been added in this release and accompanies eligible experiments e.g. [E-GEOD-125970](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-125970/results/anatomogram)
+- **Inferred cell type marker gene heatmap** - for experiments where inferred cell type data is available you can now view marker genes based on the inferred cell type clusters e.g. [E-GEOD-227038](https://www.ebi.ac.uk/gxa/sc/experiments/E-GEOD-227038/results/marker-genes)
+
+#### New experiments
+Highlights
+- **Externally analysed data** - for the first time in this release we have included a new class of data to Single Cell Expression Atlas - externally analysed data, marked with an 'E' icon.
```
Comment (irisdianauy) [inline release-notes/sc/_posts/2024-09-14-21.md]: Adding punctuations to clarify sentence
```suggestion
- **Externally analysed data** - for the first time, in this release, we have included a new class of data to Single Cell Expression Atlas - externally analysed data, marked with an 'E' icon. 
```

**Review by irisdianauy — COMMENTED** [review]
LGTM, aside from the small request to add punctuations.

**Review by sfexova — APPROVED** [review]
Made a few edits and incorporated other ppl's feedback, looks good to me now

File: release-notes/sc/_posts/2024-09-14-21.md (hunk 6 above)
Comment (sfexova) [reply release-notes/sc/_posts/2024-09-14-21.md]: thank you, I re-phrased the sentence for greater clarity

**Pull request merged**